broker/alert_ingestion/consume.py

```diff
@@ -156,8 +189,44 @@ def close(self) -> None:
         log.info(f'Closing consumer: {self.__repr__()}')
         super().close()
 
+    @staticmethod
+    def fix_schema(temp_file: TempAlertFile, survey: str, version: float) -> None:
+        """ Rewrites the temp_file with a corrected schema header
+            so that it is valid for upload to BigQuery.
+
+        Args:
+            temp_file: Temporary file containing the alert.
+            survey: Name of the survey generating the alert.
+            version: Survey version.
+        """
+
+        # get the corrected schema, if it exists
+        try:
+            f = f'valid_schemas/{survey}_v{version}.pkl'
+            with open(Path(__file__).resolve().parent / f, 'rb') as file:
+                valid_schema = pickle.load(file)
+
+        except FileNotFoundError:
+            return
```

Comment:
I would expect this to raise a warning. If the alert is not formatted correctly it may or may not raise an error when we upload to BQ (depending on the schema version and survey), but either way it would be better to catch this early in the log. 

`log.error(f'No formatted schema available for {survey} version {version}`)